### PR TITLE
Feat/lesq 1017/hubspot form submit [LESQ-1017]

### PR DESCRIPTION
**Pull request by njblore**
## Description

Music year: {owa_music_year}

- set up the onboarding form env vars
- create helper fn and types for generating onboarding hubspot form
- submit the form on completion of onboarding

## How to test

1. Go to https://deploy-preview-2705--oak-web-application.netlify.thenational.academy/onboarding
2. Click on yes / no
3. Complete the form and click continue
4. You should see your form is submitted to hubspot [here](https://app-eu1.hubspot.com/submissions/24947711/form/50399960-674f-4e23-ad66-367fbdbcdbfb/submissions/) containing the correct values

## Checklist

- [ ] The school picker form should submit correctly
- [ ] The role selection form should submit correctly

### CODE REVIEW

File: src/browser-lib/hubspot/forms/getHubspotFormPayloads.ts [hunk 1]
```diff
@@ -113,3 +113,93 @@ export const getHubspotDownloadsFormPayload = (props: {
...
+  u: unknown,
+): u is OnboardingUKTeacherProps => {
+  return (
+    typeof u === "object" &&
+    u !== null &&
+    "school" in u &&
+    "schoolName" in u &&
+    typeof (u as OnboardingUKTeacherProps).school === "string" &&
+    typeof (u as OnboardingUKTeacherProps).schoolName === "string"
+  );
```
Comment (carlmw) [inline src/browser-lib/hubspot/forms/getHubspotFormPayloads.ts:131]: Could this be done more declaratively with Zod and safeParse?

**Review by carlmw — APPROVED** [review]
Just a question about using zod instead of a big conditional. Thanks for taking care of hooking up the final form submit — I was just about to do this to flip the onboarded toggle in Clerk 🙇🏻

**Review by k-huggs — APPROVED** [review]

File: src/browser-lib/hubspot/forms/getHubspotFormPayloads.ts (hunk 1 above)
Comment (njblore) [reply src/browser-lib/hubspot/forms/getHubspotFormPayloads.ts:131]: yesss now why didn't I think of that 😅

**Review by jack-skerrett-oak — APPROVED** [review]

**Pull request merged**